src/CachePlugin.php

```diff
@@ -69,15 +69,42 @@ public function handleRequest(RequestInterface $request, callable $next, callabl
         $cacheItem = $this->pool->getItem($key);
 
         if ($cacheItem->isHit()) {
-            // return cached response
             $data = $cacheItem->get();
-            $response = $data['response'];
-            $response = $response->withBody($this->streamFactory->createStream($data['body']));
+            if (isset($data['expiresAt']) && time() < $data['expiresAt']) {
```

Comment:
can you add a code comment for the isset please? its a bit confusing when not thinking about BC with existing cache items